toptal/picasso
# feat(PageTopBar): [SCR-2646] Page TopBar add custom logo

[SCR-2646](https://toptal-core.atlassian.net/browse/SCR-2646)

### Description

Allows customizing `Page.TopBar` component for the custom service logo with `logoType` prop

### How to test

- Visit [page component](https://picasso.toptal.net/scr-2646-page-topbar-add-custom-logo/?path=/story/layout-page--page)
- Click on `Page.Footer`
- Action: Check `logoType` property
![image](https://user-images.githubusercontent.com/1483747/153020783-a6b8661f-037f-4f1c-a313-f0468b19e1f8.png)
- Open [example](https://picasso.toptal.net/scr-2646-page-topbar-add-custom-logo/?path=/story/layout-page--page#with-topscreen-logo)
- Action: Check how you can use TopScreen logo instead of default one
![image](https://user-images.githubusercontent.com/1483747/153021210-af6ffee6-54c9-41aa-8339-c26caca78d2a.png)

### Screenshots
![image](https://user-images.githubusercontent.com/1483747/153021210-af6ffee6-54c9-41aa-8339-c26caca78d2a.png)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
